app/services/webhook_endpoints/create_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module WebhookEndpoints
+  class CreateService < BaseService
+    def initialize(organization:, params:)
+      @organization = organization
+      @params = params
+
+      super
+    end
+
+    def call
+      webhook_endpoint = organization.webhook_endpoints.new(
+        webhook_url: params[:webhook_url],
+      )
+
+      webhook_endpoint.save!
```

Comment:
Don't we need to track the creation of a webhook endpoint on Segment as well?

Reply:
I'm adding it.